public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -55,6 +55,7 @@ export class PrometheusDatasource extends DataSourceApi<PromQuery, PromOptions>
   httpMethod: string;
   languageProvider: PrometheusLanguageProvider;
   resultTransformer: ResultTransformer;
+  customParameters: any;
```

Comment:
Name should reflect relevance for queries, e.g., `customQueryParameters`.

Reply:
variable renamed